examples/Potentials and capacities.ipynb

```diff
@@ -0,0 +1,671 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Stack design"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This example notebook shows how to supply parameters to a TEA class, how to plot mass- and volume loadings, how to estimate energy densities and how to loop over different parameter settings."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Let's generate TEA classes and loading plots\n",
+    "\n",
+    "A good way to create an electrode stack cell parameter-set is to start with a base parameter-set, containing the desired active materials, and update parameter such as densities with inputs for the TEA class.\n",
+    "\n",
+    "It is also possible to create parameter-sets from the bottom-up, e.g. for non-lithium chemistries, however then the key benefit of the library: PyBaMM may not work with the parameter-set. PyBaMM parameter-sets can also directly be used, for the loadings plot usally active material- and electrolyte densities are necessary inputs though.\n",
+    "\n",
+    "Below one \"design\" input-set, one \"chemistry\" input-set and one \"only_electrolyte_density\" input-set are defined to create four TEA classes, one with the \"Chen2020\" parameter-set, one without a base parameter-set and two classes having the same \"design\" input-set, but different active material related base parameter-sets."
```

Comment:
can you explain a bit more about why and how you do this? might be better to split the following code block into four so that you can add text explaining each one